development.mysql.yml

```diff
@@ -33,7 +33,7 @@ services:
       - mdb
     volumes:
       - ./notarysql/mysql-initdb.d:/docker-entrypoint-initdb.d
-    image: mariadb:10.1.28
+    image: mariadb:10.4
```

Comment:
@justincormack PTAL: do you want the image to be pinned by minor version? (and including `-bionic` suffix)?

Reply:
Its a dev image, that seems fine.